pdfminer/ccitt.py

```diff
@@ -15,6 +15,8 @@
 
 import six  #Python 2+3 compatibility
 
+import unittest
```

Comment:
What is this needed for? Was it included by mistake?

Reply:
This should be moved in the `main()` and/or the main removed entirely and moved to some testing script outside of the main library code